### ISSUE TITLE
After upgrade "undefined is not an object"

### ISSUE THREAD

**Issue by kuhbaar** [wkh237/react-native-fetch-blob#128]
Hi,

After executing "npm install" my react-native-fetch-blob got upgraded apparently:
>npm install
> XXXApp@0.0.1 E:\Development\XXX
> `-- react-native-fetch-blob@0.7.5

Starting the app now results in:
>undefined is not an object (evaluating '_reactNativeFetchBlob2.default.polyfill.Blob')

on the line:
> const Blob = RNFetchBlob.polyfill.Blob

Note: I haven't changed any links, and double checked them - the entries in settings.gradle and mainActivity.java are still there.

Any ideas? I'm using react-native v0.31

**Comment by wkh237** [comment]
@kuhbaar , looks like you're using polyfills in `0.7.5`, but they're available in `0.8.0+`, please check [our wiki](https://github.com/wkh237/react-native-fetch-blob/wiki/Web-API-Polyfills-(experimental)#user-content-blob). 

You may have to upgrade the library you've installed by the following command 

```
$ npm install --save react-native-fetch-blob
$ react-native link
```

**Comment by kuhbaar** [comment]
Updated to newest version and it works now. Wonder why it was set to 0.7.0 though
